docs/user/faqs.rst

```diff
@@ -84,11 +84,17 @@ For OS X::
     $ brew install mecab-ipadic
 
 Afterwards, you can use ``pip install fonduer[spacy_ja]`` to install Fonduer
-with Japanese language support. If you would like to use other languages with
-spaCy alpha support, which are not yet integrated in Fonduer, feel free to open
-an Issue_.
+with Japanese language support.
+
+If you would like to use Fonduer for Chinese documents, you can use
+``pip install fonduer[spacy_ja]`` to install Fonduer with Chinese language support.
```

Comment:
This description is looks wrong? Should be `spacy_zh`? Both Chinese and Japanese should be described.

Reply:
Fixed! Thanks!